randovania/games/am2r/logic_database/Main Caves.txt

```diff
@@ -1129,13 +1129,13 @@ Extra - minimap_data: [{'x': 42, 'y': 26}, {'x': 43, 'y': 26}, {'x': 44, 'y': 26
           # Hang off first platform
           Power Grip and Movement (Beginner)
 
-> Area Transition to Research Site Elevator; Heals? False
+> Horizontal Dock to Research Site Elevator; Heals? False
```

Comment:
i wonder whether we should make these a new type? they're at least a unique door.

Reply:
would be useful as a) more door variety b) a fully trivial door in the case we ever go back to the concept of making power beam an item.

Reply:
could also just slap a todo and call it a day if we wanna have that discussion for another day

Reply:
Do we really need more doors? I'll concede to the TODO for later discussion.